### PR TITLE
feat(); Uniswap V3 Base; Add Uniswap V3 Base

### PR DESCRIPTION
**Pull request by steegecs**
**Context:**
Uniswap V3 has been deployed on the Base network. This PR adds the Base network deployment to the Uniswap V3 Forks library. 

https://twitter.com/Uniswap/status/1688581784609435648

**Note:**
There is not very good coverage of the Base network chain yet, and it looks like on the Base Uniswap Analytics page, it is not very clear which pools are the big players, and if there is really much TVL at all. This makes it hard to select tokens for the whitelist and the stable oracle pools. I only added 3 whitelist tokens, USDC as the stablecoin, and a few pools with different fee percentages paired with wETH. We'll probably need to go back in and update the pricing parameters once there is more traction to gain wider coverage and more reliability.

### CODE REVIEW

File: subgraphs/uniswap-v3-forks/src/common/constants.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export namespace Network {
   export const AVALANCHE = "AVALANCHE";
   export const AURORA = "AURORA";
   export const BSC = "BSC"; // aka BNB Chain
+  export const BASE = "BASE"; // aka Ethereum mainnet
```
Comment (dhruv-chauhan) [inline subgraphs/uniswap-v3-forks/src/common/constants.ts:16]: nit: remove comment

**Review by dhruv-chauhan — APPROVED** [review]
lgtm!

File: deployment/deployment.json [hunk 2]
```diff
@@ -3679,11 +3679,33 @@
           }
         }
       },
+      "uniswap-v3-base": {
+        "network": "base",
```
[the remark this answers is not in the document]
Comment (dmelotik) [reply deployment/deployment.json:3683]: Screen Shot 2023-08-08 at 1 13 56 PM

[the remark this answers is not in the document]
Comment (dmelotik) [reply deployment/deployment.json:3683]: I would say it's fine. We have a subgraph for aave v3 metis that is not supported by the graph either.

I think it is important to be accurate in that json object imo, whether the graph supports it or not is a separate issue

File: subgraphs/uniswap-v3-forks/src/common/constants.ts (hunk 1 above)
Comment (steegecs) [reply subgraphs/uniswap-v3-forks/src/common/constants.ts]: Removed.

File: deployment/deployment.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (steegecs) [reply deployment/deployment.json:3683]: Yeah, we can actually already deploy to hosted service. Docs not updated yet. 

https://thegraph.com/hosted-service/subgraph/messari/uniswap-v3-base

**Pull request merged**